internal/queries/main.go

```diff
@@ -44,3 +46,14 @@ func runQuery(query string, client *bigquery.Client) (*bigquery.RowIterator, err
 	q := client.Query(query)
 	return q.Read(ctx)
 }
+
+func getTitleField(ledgerID, closedAt, aggregateBy string) string {
```

Comment:
Could you add comments with what each clause would look like? This made sense to me after I read it over a couple of times, which signals that a comment would be helpful.